### ISSUE TITLE
historyApiFallback doesn't work

### ISSUE THREAD

**Issue by Guria** [Yomguithereal/kotatsu#113]
Tried to use [historyApiFallback option](http://webpack.github.io/docs/webpack-dev-server.html#the-historyapifallback-option) in my webpack config, but it seems doesn't work:
```js
{
  devServer: {
    historyApiFallback: true
  },
  module: { /* ...*/ }
}
```

**Comment by Yomguithereal** [comment]
This should have been fixed by #110, I guess.

**Comment by Guria** [comment]
Yeap, just noted it.
